### PR TITLE
filtering of 'Suggestions'

### PR DESCRIPTION
**Pull request by jakub-trusina**
Hey E2rd,

I've added filtering out of "Suggestions" which appeared lately. I'm not sure about English or Czech versions. I haven't tested it though as I am using Firefox and don't know how to test it there.

Best regards,
Kuba

### CODE REVIEW

File: launcher.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const LANG = {
     "en": {
         "Sponsored": "Sponsored",
         "Sponsored · Paid for by": "Sponsored · Paid for by",
+        "Suggestions": "Suggestions",
```
Comment (e3rd) [inline launcher.js:8]: The key name seems french to me, change to "Suggested" (?)

File: launcher.js [hunk 2]
```diff
@@ -63,6 +66,9 @@ const is_garbage = n => {
         // xlink -> <use -> "S"> <use -> "ponsored">
         return true
     }
+    if (n.textContent.startsWith(lang["Suggested"]) || n.textContent.startsWith(lang["Suggested"].substr(1))) {
+        return true
```
Comment (e3rd) [inline launcher.js:70]: Have you encountered the need of `.substr(1)`? If not, put it rather below to the block:
```
if (!n.children.length) {
        if ([lang["Suggested for you"],
        lang["Suggested live gaming broadcast"],
        lang["Suggested groups"],
        lang["People You May Know"],
        lang["Friend Requests"],
        lang["Videos Just For You"]]
```

**Review by e3rd — CHANGES_REQUESTED** [review]
review :)

File: launcher.js (hunk 1 above)
Comment (jakub-trusina) [reply launcher.js:8]: Yes. It sounds French. But translation of Suggestions to English is "Suggestions". :D

File: launcher.js (hunk 2 above)
Comment (jakub-trusina) [reply launcher.js:70]: No. I'll update it

**Review by e3rd — CHANGES_REQUESTED** [review]

File: launcher.js [hunk 3]
```diff
@@ -74,7 +77,8 @@ const is_garbage = n => {
         lang["Suggested groups"],
         lang["People You May Know"],
         lang["Friend Requests"],
-        lang["Videos Just For You"]]
+        lang["Videos Just For You"]],
+        lang["Suggested"]
```
[the remark this answers is not in the document]
Comment (e3rd) [reply launcher.js]: You still used `lang["Suggested"]` místo svýho klíče Suggestions, vzdávám to :D

**Pull request merged**